main.html

```diff
@@ -0,0 +1,126 @@
+            <ul>
+                <li id="home"><a href="index.html">Home</a></li>
+                <li id="aboutus"><a href="main.html">About Us</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="program"><a href="#">Program</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="join"><a href="#">Join</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="sponsor"><a href="#">Sponsor</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="news"><a href="#">News</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="campaign"><a href="#">CC Campaign</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+            </ul>
+        </div>
+    </header>
+    <main class="mainContainer container-fluid">
+        <div id="aboutUs" class="aboutus container">
+            <h3 class="aboutusgreetings">"Hello, Sharing World"</h3>
+            <p class="main aboutustitle">Creating Global Video Forums Regarding The Newest News On Videogames</p>
+            <p class="aboutusparagraph">The global videogames forum brings together thousnand of people that loves playing videogames and for it we have decided to dedicate out time to bring that to the workd's eyes with mega events like this.</p>
+            <p class="aboutusparagraph">We count wiht the support of very immportant industries on the world like Play Station, Nintendo and Epic Games, they all share their new projects and latest news on their franquicies and what is coming up next.</p>
+            <p class="aboutusparagraph">Please contact us per email for any further questions about Global Video Forums.
+            <p class="aboutusparagraph">hectorlau.1993@gmail.com</p>
+        </div>
+        <div class="creative container">
+            <h4>Creating Global Video Forums Logo</h4>
+            <p>The logo of Creating Global Video Forums was decided through the logo competition from the first of February to the 28th of the same month. After online public voting and selection commmittee's judging process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimized the prize-winning-logo fro this year's summit as following.</p>
+            <div class="companiesLogo3">
+                <a href="index.html"><img class="logoIcon" src="./images/first_page/logo.jpg" alt=""></a>
+                <p>Global Video Games</p>
+            </div>
+        </div>
+        <div class="past container">
+            <h4>See the past Creating Global Video Forums</h4>
+            <p>Take a loog at the previous works we have done so far and search for more inforamtion about how this events ara alike.</p>
+            <div>
```

Comment:
- [ ] This section is currently missing the margins and paddings as shown in the template, please make sure to differentiate the different sections to better match the template. Make sure to also remove the background image from the sections that shouldn't have it.


| Your website  | Template |
| ------------- | ------------- |
| ![](https://i.imgur.com/lcPm7F9.png)  | ![](https://i.imgur.com/EQLVxlg.png)  |